pkg/scaffold/olm-catalog/csv.go

```diff
@@ -147,10 +152,14 @@ func (s *CSV) getBaseCSVIfExists() (*olmapiv1alpha1.ClusterServiceVersion, bool,
 	if s.FromVersion != "" {
 		verToGet = s.FromVersion
 	}
-	lowerProjName := strings.ToLower(s.ProjectName)
-	name := getCSVFileName(lowerProjName, verToGet)
-	fromCSV := filepath.Join(s.pathPrefix, scaffold.OLMCatalogDir, lowerProjName, verToGet, name)
-	return getCSVFromFSIfExists(s.getFS(), fromCSV)
+	csv, exists, err := getCSVFromFSIfExists(s.getFS(), s.getCSVPath(verToGet))
+	if err != nil {
+		return nil, false, err
+	}
+	if !exists && s.FromVersion != "" && verToGet == s.FromVersion {
```

Comment:
Is it possible for `s.FromVersion != ""` and `verToGet != s.FromVersion`? On line 152, if `s.FromVersion != nil`, then `verToGet = s.FromVersion`, so I don't think we need the `verToGet == s.FromVersion` check.